zfsbootmenu/lib/zfsbootmenu-lib.sh

```diff
@@ -78,17 +78,22 @@ draw_be() {
 
   zdebug "using environment file: ${env}"
 
+  if [ -f "${BASE}/cmdline" ]; then
+    kcl_text="[CTRL+T] revert kcl"
+    kcl_bind=",alt-t"
+  fi
+
   header="$( column_wrap "\
 [RETURN] boot:[ESCAPE] refresh view:[CTRL+P] pool status
 [CTRL+D] set bootfs:[CTRL+S] snapshots:[CTRL+K] kernels
 [CTRL+E] edit kcl:[CTRL+J] jump into chroot:[CTRL+R] recovery shell
-[CTRL+L] view logs::[CTRL+H] help" \
+[CTRL+L] view logs:${kcl_text}:[CTRL+H] help" \
```

Comment:
I propose rearranging this line:
```suggestion
${kcl_text}:[CTRL+L] view logs:[CTRL+H] help" \
```
This will leave the hole on the left by default, but that hole is right below the KCL edit option so they will be nearby.

Alternatively:
```sh
[CTRL+E] edit kcl:${kcl_text}:[CTRL+J] jump into chroot
[CTRL+R] recovery shell:[CTRL+L] view logs:[CTRL+H] help" \
```
That makes the last row "recovery/debug" related and leaves the blank right next to the KCL edit.

Reply:
I dropped screenshots of this on IRC. I don't love it or hate it. If you have strong feelings one way or the other, I"ll make the layout match whatever you prefer.

Reply:
Settled on shifting `[CTRL+L] view logs` over into the empty space only when the revert option is available. This preserves the existing ITEM1 SPACE ITEM2 layout on the bottom in the normal case, becoming ITEM3 ITEM1 ITEM2 on demand.